File: node/src/continent/imageHandler.ts
import fs from "fs";

const dirPathName = "../common/images/";

export const GetImage = (name: string) => {
    const imagePath = dirPathName + name + ".svg";

    try {
        return fs.readFileSync(imagePath, "utf8");
    } catch (error) {
        return "";
    }
};

File: go/src/continent/imageHandler.go
package continent

import (
	"net/http"
	"strings"

	"github.com/getsalty/Webservice_POC/go/src/reader"
)

var dirPathName = "../common/images/"

func GetImage(w http.ResponseWriter, r *http.Request) {

	name := strings.TrimPrefix(r.URL.Path, "/continent/image/")

	if len(name) == 0 {
		return
	}

	imagePath := (dirPathName + name + ".svg")

	byteValue, err := reader.ReadFile(imagePath)
	if err != nil {
		return
	}

	w.Write(byteValue)
}
